cluster/aws.go

```diff
@@ -783,3 +785,54 @@ func getKubicornLogLevel() int {
 		return 4
 	}
 }
+
+// ListRegions lists supported regions
+func ListRegions() ([]*ec2.Region, error) {
+	// Load session from shared config
+	sess := session.Must(session.NewSessionWithOptions(session.Options{
```

Comment:
Maybe this can be factored out getting the session and creating a new EC2 client ListAMI and ListRegion does the same thing